catalog/plugin/catalog-plugin-metacardbackup-filestorage/src/main/java/org/codice/ddf/catalog/plugin/metacard/backup/storage/filestorage/MetacardFileStorageRoute.java

```diff
@@ -64,17 +69,25 @@ public void configure() throws Exception {
             METACARD_BACKUP_KEEP_DELETED_RTE_PROP,
             simple(String.valueOf(keepDeletedMetacards), Boolean.class))
         .choice()
-        .when(not(getCheckDeletePredicate()))
-        .stop()
-        .otherwise()
-        .choice()
         .when(not(getShouldBackupPredicate()))
         .stop()
         .otherwise()
         .setHeader(
             TEMPLATED_STRING_HEADER_RTE_PROP,
             method(new MetacardTemplate(outputPathTemplate), "applyTemplate(${body})"))
         .to("catalog:metacardtransformer")
+        .choice()
+        .when(
+            and(
+                header(PostIngestConsumer.ACTION).isEqualTo(PostIngestConsumer.DELETE),
+                not(getCheckDeletePredicate())))
+        .bean(
+            MetacardFileStorageRoute.class,
+            String.format(
+                "deleteFile(%s, ${in.headers.%s)",
```

Comment:
I know we talked about this, but just to verify, I am guessing 
`
    .to(
           "file://"
                  + getStartingDir()
                 + "?fileName=${in.headers."
                 + TEMPLATED_STRING_HEADER_RTE_PROP
                 + "}&delete=true");
`
didn't work

Reply:
According to file2, adding delete to the consumer just deletes the processed file from the producer (which in this case wouldn't be the destination file).
http://camel.apache.org/file2.html